README.md

```diff
@@ -15,12 +15,14 @@ on:
     types: [opened, edited]
   issue_comment:
     types: [created, edited]
+  discussion:
+    types: [created, edited]
 
 jobs:
   accessibility_alt_text_bot:
     name: Check alt text is set on issue or pull requests
     runs-on: ubuntu-latest
-    if: ${{ github.event.issue || github.event.pull_request }}
+    if: ${{ github.event.issue || github.event.pull_request || github.event.discussion }}
```

Comment:
We should add this to our [test workflow](https://github.com/github/accessibility-alt-text-bot/blob/main/.github/workflows/test-accessibility-alt-text-bot.yml) in this repo